phpstan.neon

```diff
@@ -0,0 +1,7 @@
+parameters:
+  bootstrapFiles:
+    - %currentWorkingDirectory%/app/bootstrap.php
+    - %currentWorkingDirectory%/app/config/container.php
+
+  excludes_analyse:
+    - %currentWorkingDirectory%/src/Tests/*/*.php
```

Comment:
This excludes files, so that CI reports that only 2 files are analysed.
https://drone.owncloud.com/owncloud/updater/122/2/4